detections/endpoint/windows_scheduled_task_service_spawned_shell.yml

```diff
@@ -0,0 +1,75 @@
+name: Windows Scheduled Task Service Spawned Shell
+id: d8120352-3b62-4e3c-8cb6-7b47584dd5e8
+version: 1
+date: '2023-06-13'
+author: Steven Dick
+status: production
+type: TTP
+description: The following analytic identifies when the Task Scheduler service "svchost.exe -k netsvcs -p -s Schedule" is the parent process 
+to common command line, scripting, or shell execution binaries. Attackers often abuse the task scheduler service with these binaries as an 
+execution and persistence mechanism in order to blend in with normal Windows operations. 
+This TTP is also commonly seen for legitimate purposes such as business scripts or application updates.
+data_source:
+- Sysmon 1
+- Windows Security 4688
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes where 
+Processes.parent_process="*\\system32\\svchost.exe -k netsvcs -p -s Schedule*" Processes.process_name 

```

Comment:
When i tested this detection, it looks like `Processes.parent_process` is unknown in our environment and hence its failing testing. Can you share the screenshot of this search?

Reply:
Hrmm that's interesting. What Sysmon TA are you leveraging ? I tested this against the ["official" Sysmon TA](https://splunkbase.splunk.com/app/5709)

Here's a screencap of search in action:
![image](https://github.com/splunk/security_content/assets/38897662/254fd32c-6e8b-4220-a92c-e19cf8f92d8f)

Reply:
You'll have to forgive the masking, it's from production data.

Reply:
my bad, we saw errors earlier since the attack data file earlier had different source types. after we fixed that attack data, your detections passed gracefully!

https://github.com/splunk/security_content/pull/2766
sorry about the trouble!